courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/3_keras_sequential_api.ipynb

```diff
@@ -156,6 +156,13 @@
     "We use a python dictionary comprehension to create the feature columns for our model, which is just an elegant alternative to a for loop."
```

Comment:
<p>People will have already seen gcloud ai-platform versions create before so that they know what the parameters are? Origin is always confusing for me haha. I wish it was like "path_to_saved_model_files" or something. Maybe just a link to the documentation is good enough to save them a Google search?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>good call. They will have seen the code in the slides and I've specifically called out what 'origin' refers to bc I had the same thought lol. </p><p>I'll add a link to documentation too. </p>